lms/templates/shoppingcart/receipt.html

```diff
@@ -304,17 +304,16 @@ <h3 class="text-center">${_("Please send each professional one of these unique r
               </span>
             </h3>
 
-            <h1>${_(" {course_name} ").format(course_name=course.display_name)}
-              <span class="pull-right">
-                % if course_start_time:
-                  ${course_start_time}
-                %endif
-                -
-                % if course_end_time:
-                  ${course_end_time}
-                %endif
-              </span>
-            </h1>
+            <h1>${_(" {course_name} ").format(course_name=course.display_name)}</h1>
```

Comment:
It's really unlikely there's going to be a value for the course name in the translation dictionary. It's almost impossible that there's going to be a value for "space course name space". Why the extra spaces? And is it really the right behavior to translate the course's name?